package com.consultantvendor.data.network

const val PER_PAGE_LOAD = 20
const val PER_PAGE_LOAD_CHAT = 50


object ApiKeys {

    /*facbook,google,email,phone*/
    const val PROVIDER_TYPE = "provider_type"

    /*optional only phone and email*/
    const val PROVIDER_ID = "provider_id"

    /*access_token or password or otp*/
    const val PROVIDER_VERIFICATION = "provider_verification"
    const val USER_TYPE = "user_type"

    const val AFTER = "after"
    const val BEFORE = "before"
    const val PER_PAGE = "per_page"
}

object ProviderType {
    const val facebook = "facebook"
    const val google = "google"
    const val email = "email"
    const val phone = "phone"
}

object LoadingStatus {
    const val ITEM = 0
    const val LOADING = 1
}


object PushType {
    const val PROFILE_APPROVED = "PROFILE_APPROVED"
    const val CHAT = "chat"
    const val CHAT_STARTED = "Chat Started"
    const val NEW_REQUEST = "NEW_REQUEST"
    const val NEW_USER = "NEW_USER"
    const val BOOKING_REQUEST = "BOOKING_REQUEST"
    const val REQUEST_FAILED = "REQUEST_FAILED"
    const val CANCELED_REQUEST = "CANCELED_REQUEST"
    const val RESCHEDULED_REQUEST = "RESCHEDULED_REQUEST"
    const val UPCOMING_APPOINTMENT = "UPCOMING_APPOINTMENT"
    const val REQUEST_COMPLETED = "REQUEST_COMPLETED"
    const val PATIENT_ADDED_SYMPTOMS="PATIENT_ADDED_SYMPTOMS"
    const val COMPLETED = "COMPLETED"
    const val AMOUNT_RECEIVED = "AMOUNT_RECEIVED"
    const val PAYOUT_PROCESSED = "PAYOUT_PROCESSED"
    const val PAYOUT_FAILED="PAYOUT_FAILED"
    const val ASSINGED_USER = "ASSINGED_USER"
    const val BALANCE_ADDED = "BALANCE_ADDED"
    const val BALANCE_FAILED="BALANCE_FAILED"

    const val CALL_RINGING = "CALL_RINGING"
    const val CALL_ACCEPTED = "CALL_ACCEPTED"
    const val CALL_CANCELED = "CALL_CANCELED"
    const val FREE_EXPERT_ADVISE = "FREE_EXPERT_ADVISE"
    const val PAID_EXTRA_PAYMENT="PAID_EXTRA_PAYMENT"
//    const val REQUEST_LOGIN_ACCEPTED = "REQUEST_LOGIN_ACCEPTED"
//    const val CANCELED_LOGIN_REQUEST = "CANCELED_LOGIN_REQUEST"
}